### DISCUSSION TITLE
Certificate verification failed

### DISCUSSION THREAD

**Discussion by cyrakuse** [oerdnj/deb.sury.org#1900] [Q&A]
I've installed a fresh Debian 11.
i've installed the repo.
When i make a "apt update", i obtain this message : 

_Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Erreur de vérification du certificat. [IP : 185.93.2.243 443]_

any idea ?

**Comment by thomaszbz** [comment]
I filed a new issue which analyzes the problem: https://github.com/oerdnj/deb.sury.org/issues/1943
